powerauth-nextstep-client/src/main/java/io/getlime/security/powerauth/lib/nextstep/client/NextStepClient.java

```diff
@@ -199,6 +200,88 @@ private RestTemplate defaultTemplate() {
+     *
+     * @param userId User ID
+     * @return List of enabled authentication methods for given user wrapped in GetAuthMethodResponse.
+     * @throws NextStepServiceException Exception with {@link ErrorModel} for ERROR status
+     */
+    public Response<GetAuthMethodsResponse> getAuthMethodsEnabledForUser(String userId) throws NextStepServiceException {
+        try {
+            // Exchange next step request with NextStep server.
+            ResponseEntity<Response<GetAuthMethodsResponse>> response = defaultTemplate().exchange(serviceUrl + "/user/" + userId + "/auth-method", HttpMethod.GET, null, new ParameterizedTypeReference<Response<GetAuthMethodsResponse>>() {
+            });
+            return new Response<>(Response.Status.OK, response.getBody().getResponseObject());
+        } catch (HttpStatusCodeException ex) {
+            throw handleHttpError(ex);
+        } catch (ResourceAccessException ex) {
+            throw handleResourceAccessError(ex);
+        }
+    }
+
+    /**
+     * Enable an authentication method for given user.
+     *
+     * @param userId     User ID
+     * @param authMethod Authentication method
+     * @return List of enabled authentication methods for given user wrapped in GetAuthMethodResponse.
+     * @throws NextStepServiceException Exception with {@link ErrorModel} for ERROR status
+     */
+    public Response<GetAuthMethodsResponse> enableAuthMethodForUser(String userId, AuthMethod authMethod) throws NextStepServiceException {
+        try {
+            // Exchange next step request with NextStep server.
+            ResponseEntity<Response<GetAuthMethodsResponse>> response = defaultTemplate().exchange(serviceUrl + "/user/" + userId + "/auth-method/" + authMethod, HttpMethod.POST, null, new ParameterizedTypeReference<Response<GetAuthMethodsResponse>>() {
+            });
+            return new Response<>(Response.Status.OK, response.getBody().getResponseObject());
+        } catch (HttpStatusCodeException ex) {
+            throw handleHttpError(ex);
+        } catch (ResourceAccessException ex) {
+            throw handleResourceAccessError(ex);
+        }
+    }
+
+    /**
+     * Disable an authentication method for given user.
+     *
+     * @param userId     User ID
+     * @param authMethod Authentication method
+     * @return List of enabled authentication methods for given user wrapped in GetAuthMethodResponse.
+     * @throws NextStepServiceException Exception with {@link ErrorModel} for ERROR status
+     */
+    public Response<GetAuthMethodsResponse> disableAuthMethodForUser(String userId, AuthMethod authMethod) throws NextStepServiceException {
+        try {
+            // Exchange next step request with NextStep server.
+            ResponseEntity<Response<GetAuthMethodsResponse>> response = defaultTemplate().exchange(serviceUrl + "/user/" + userId + "/auth-method/" + authMethod, HttpMethod.DELETE, null, new ParameterizedTypeReference<Response<GetAuthMethodsResponse>>() {
```

Comment:
Please do not include userId in URL, favor request object with userId as an attribute. (Using DELETE method is OK, since it allows requests to have body and hence there is not the issue with query attributes / sensitive data in URL).